### PR TITLE
68 todo move constraints from create postjs to new js files

### PR DESCRIPTION
**Pull request by agctute**
all constraints should be added to relevant js files including editPost.js. Also added character count to post creation pages.

### CODE REVIEW

File: scheduler/create_post/createFb.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
+const summary = document.getElementById('post-summary');
 const postDescription = document.getElementById('desc-input');
-const postTag = document.getElementById('tag');
+const date = document.getElementById('date-to-post');
+const time = document.getElementById('time-to-post');
```
Comment (aolsen07) [inline scheduler/create_post/createFb.js:4]: dateElement and timeElement?

Comment (agctute) [reply scheduler/create_post/createFb.js:4]: one is dd-mm-yyyy, other is hh/mm AM/PM

**Comment by MathElderGod** [comment]
bug found, where edit post click, the post gets removed from local storage. was not an issue before. Recommend not merging until further investigation.

**Comment by MathElderGod** [comment]
this branch has some code-breaking bugs. I and Kevin tried to fix them, but we don't understand why the bugs are happening. They're mainly tied to the char limit. Some variables were never defined, and there was a duplicated event listener, which was a bit confusing. In the meantime, I recommend this branch be continuously tested to ensure all specifications are met as desired....because currently, this does not work.

**Comment by Yr-Nemsis** [comment]
Also, the path should be kept absolute after testing (Not relative such as ../assets/create-post-background.png) to avoid any problem caused by relative url

**Comment by MathElderGod** [comment]
i think this is okay, but I think I am applying a couple of fixes

**Comment by agctute** [comment]
> Also, the path should be kept absolute after testing (Not relative such as ../assets/create-post-background.png) to avoid any problem caused by relative URLs

I'd prefer if we used relative URLs instead, since it's a standard industry practice to only use relative URLs, for example [this post](https://stackoverflow.com/questions/2005079/absolute-vs-relative-urls) talks about it:

The main issue is that for larger projects, an individual making a branch would have to change all the URLs every time they want to branch and test something, which is what we do. You can see that if there were hundreds of URLs, this can quickly become extraordinarily time-consuming and prone to mistakes. If there are any other questions regarding this you can DM me about it.

**Comment by agctute** [comment]
closed for further testing

**Pull request closed without merging**